README.md

```diff
@@ -0,0 +1,18 @@
+# FC calibration tool
+
+This is a tool that helps calibrate INAV FCs.
+
+## Installation
+
+To install and run this tool, you need to clone this repository and all its submodules:
+
+```
+$ git clone --recursive https://github.com/shellixyz/fc_calibration_tool.git
+$ sudo gem install serialport crc
```

Comment:
Now that I added a Gemfile you can simply do `bundle install` to install all the dependencies. It also depends on nokogiri for parsing Mavlink messages definitions stored in XML files